rust/connlib/shared/src/messages/client.rs

```diff
@@ -88,8 +92,7 @@ pub struct ResourceDescriptionInternet {
 impl ResourceDescriptionInternet {
     pub fn with_status(self, status: Status) -> crate::callbacks::ResourceDescriptionInternet {
         crate::callbacks::ResourceDescriptionInternet {
-            name: "Internet Resource".to_string(),
-            address: "All internet addresses".to_string(),
+            name: self.name.unwrap_or("○ Internet Resource".to_string()),
```

Comment:
This is the same character we use to show "No activity" for Resource status I believe. How about this instead?

```suggestion
            name: self.name.unwrap_or("<-> Internet Resource".to_string()),
```